code/modules/planet/sif.dm

```diff
@@ -330,7 +332,7 @@ datum/weather/sif
 			if(amount_soaked >= 10)
 				continue // No need to apply damage.
 
-			H.apply_damage(rand(5, 10), BRUTE, target_zone, amount_blocked, amount_soaked, used_weapon = "hail")
+			H.apply_damage(rand(1, 5), BRUTE, target_zone, amount_blocked, amount_soaked, used_weapon = "hail")
 			to_chat(H, "<span class='warning'>The hail smacks into you!</span>")
```

Comment:
It might be worth putting an ``if(prop(some_number))`` statement here to reduce the chatlog getting filled with hailspam.

Reply:
An excellent catch, but since rain and storms do the same thing, I'm setting up a timer instead.